gtars/src/common/models/region_set.rs

```diff
@@ -66,10 +68,26 @@ impl TryFrom<&Path> for RegionSet {
                 | string_line.starts_with("#")
             {
                 header.push_str(&string_line);
+                first_line = false;
                 continue;
             }
 
-            // println!("parts: {:?} -- {:?}", parts, string_line);
+            // Handling headers like `chr start end etc` without #
```

Comment:
what about multiple lines? And lines that do begin with `#`?

Reply:
we already have it. It is being checked before this check

Reply:
Ok, so this is really a 2nd check, and it is specifically for a _column header_.
```suggestion
            // Handling column headers like `chr start end etc` without #
```

Reply:
Curious, do we have an idea of what percentage of files this is occurring for that we have encountered so far?